pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -5627,4 +5639,591 @@ def prepare_plots(self):
             self.plot_dicts[plot_name].update(
                 {'legend_ncol': 2,
                  'legend_bbox_to_anchor': (1, -0.15),
-                 'legend_pos': 'upper right'})
\ No newline at end of file
+                 'legend_pos': 'upper right'})
+
+class MultiQutrit_Timetrace_Analysis(ba.BaseDataAnalysis):
```

Comment:
Why is this not in readout_analysis?

Reply:
I could move it there, no strong opinion

Reply:
Though I cannot do it for the multiqutrit ssro analysis because it is based on multiqubit timedomain (a bit unfortunate from a semantic point of view I admit but I would otherwise duplicate big chuncks of code) and importing tda in readout analysis would create a circular import (readout analysis is already imported in timedomain analysis)

Reply:
I see. I don't have a strong preference, we can leave them in tda. I just wasn't sure why you chose to implement them there.